### DISCUSSION TITLE
Where do I find system infos

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32346] [Q&A] [answered]
I am looking for a link to system infos in beta 7. Where do I find this link?

**Comment by Quy** [comment, marked as answer]
Go to the System dashboard.

![sys-info](https://user-images.githubusercontent.com/368084/107150470-c9bf0900-6912-11eb-9efb-96c44c4c8395.png)

**Reply by richard67** [reply]
@agi-code Does that answer your question?
